docs/references/keywords.rst

```diff
@@ -183,8 +183,7 @@ extensions).
 
 ``data_files``
     .. warning::
-        ``data_files`` is deprecated. It does not work with wheels, so it
-        should be avoided.
+        ``data_files`` is deprecated.
```

Comment:
Should we then say something like the following:

```
This is an advanced feature and it is *not intended to work with absolute installation paths*.
All files listed in ``data_files`` will be installed in paths relative to a directory
decided by the package installer (e.g. `pip`),
which usually results in nesting under a virtual environment.
See :docs:`userguide/datafiles` for an alternative placing inside the package directory.
Please do not use this setting for things like man-pages, application launchers or anything that requires system-wide installation.
```

Reply:
I updated the warning with this